src/bundle/Resources/config/configuration.yaml

```diff
@@ -0,0 +1,31 @@
+imports:
+    - { resource: ui/mappers.yaml }
+
+parameters:
+    # set via compiler pass
+    ezplatform.ezrichtext.alloy_editor: []
+
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    EzSystems\EzPlatformRichText\Configuration\Provider\:
```

Comment:
I'd remove this definition as it's too greedy. Also you are registering explicitly all classes under `EzSystems\EzPlatformRichText\Configuration\Provider` namespace below.

Reply:
> I'd remove this definition as it's too greedy. Also you are registering explicitly all classes under `EzSystems\EzPlatformRichText\Configuration\Provider` namespace below.

Right, I thought AlloyEditor was still configured by this, but I see now that it's explicit as well. Fixed via b333cb1fdd31730e6d084370ca570f98ad593879